internal/pkg/service/stream/keboolasink/bridge/file.go

```diff
@@ -63,7 +65,11 @@ func (b *Bridge) createStagingFile(ctx context.Context, now time.Time, sink defi
 		ctx,
 		attribute.String("token.ID", token.Token.ID),
 		attribute.String("file.name", name),
-		attribute.String("file.key", file.FileKey.String()),
+		attribute.String("project.id", file.ProjectID.String()),
+		attribute.String("branch.id", file.BranchID.String()),
+		attribute.String("source.id", file.SourceID.String()),
+		attribute.String("sink.id", file.SinkID.String()),
+		attribute.String("file.id", file.FileID.String()),
 	)
```

Comment:
Finally, the `file.Telemetry()` method should be used here:

```
attrs := file.Telemetry()
attrs = appent(attrs, attribute.String("token.ID", token.Token.ID), ...)
ctx = ...
```

Martin has some code prepared, so this can be finalized later.